src/main/java/com/epam/tc/hw9api/constants/Constants.java

```diff
@@ -0,0 +1,7 @@
+package com.epam.tc.hw9api.constants;
+
+public class Constants {
+    public static String BOARDS_ENDPOINT = "/boards/";
+    public static String LISTS_ENDPOINT = "/lists/";
+    public static final String MEMBER_ENDPOINT = "/members/";
```

Comment:
move  to BoardSteps, since you only use MEMBER_ENDPOINT there

Reply:
moved